airbyte-integrations/bases/debezium-v1-9-2/src/main/java/io/airbyte/integrations/debezium/internals/PostgresConverter.java

```diff
@@ -21,7 +22,7 @@ public class PostgresConverter implements CustomConverter<SchemaBuilder, Relatio
 
   private static final Logger LOGGER = LoggerFactory.getLogger(PostgresConverter.class);
 
-  private final String[] DATE_TYPES = {"DATE", "DATETIME", "TIME", "TIMETZ", "INTERVAL", "TIMESTAMP", "TIMESTAMPTZ"};
```

Comment:
What would happen to `DATETIME` types now after it is removed from this array?

Reply:
Hah! this was probably a left over from a copy paste I guess. Postgres doesn't have a DATETIME data type
https://www.postgresql.org/docs/current/datatype-datetime.html

Reply:
Got it. Cool.